tools/walletextension/test/wallet_extension_test.go

```diff
@@ -70,14 +92,15 @@ func createWalExt() (func(), error) {
 
 // Creates a dummy host that the wallet extension can connect to.
 func createDummyHost() (*http.Server, error) {
-	server := &http.Server{Addr: fmt.Sprintf("%s:%d", localhost, nodePortWS)} //nolint:gosec
-	http.HandleFunc("/ready", func(w http.ResponseWriter, r *http.Request) {})
-	http.HandleFunc("/", func(w http.ResponseWriter, r *http.Request) {
+	serveMux := http.NewServeMux()
```

Comment:
This approach was required to allow us to run multiple tests from the same file. Otherwise, it complains about endpoint reregistration.